docs/zh_CN/NNICTLDOC.md

```diff
@@ -360,7 +360,8 @@ nnictl 支持的命令：
   
   | 参数及缩写 | 是否必需  | 默认值 | 说明         |
   | ----- | ----- | --- | ---------- |
-  | id    | False |     | Trial 的 id |
+  | id         | False |     | 需要设置的 Experiment 的 id. 当有多个 Experiments 时，需要填写该参数 |
```

Comment:
@squirrelsc  I assume we do not need to update Chinese version?

Reply:
Update it in crowd in later, after this submitted. Don't change Chinese directly.

Reply:
Sorry. I discard this change to Chinese doc.